projects/plugins/jetpack/extensions/blocks/podcast-player/deprecated/v1/index.js

```diff
@@ -9,7 +8,6 @@ import colorValidator from '../../../../shared/colorValidator';
 const attributes = {
 	url: {
 		type: 'string',
-		validator: isUrl,
```

Comment:
Wouldn't this be a behavior change? Also, I do see `isUrl` in `@wordpress/url`. 🤔

Reply:
Yeah, but I didn't see `isUrl` anywhere in `@wordpress/url`. Now that you claimed you saw it, I still don't see it but I do see a `isURL`.

Since there's no `isUrl`, I think this wound up being equivalent to `validator: undefined`, and apparently no one noticed that no validation was happening.

Reply:
Ah, indeed, it's `isURL` on my end as well. At least I did see _something_. 😆